Cataclysm/APLs/RogueAssasination.simc

```diff
@@ -11,19 +11,19 @@ actions+=/variable,name=regen_saturated,value=energy.regen_combined>settings.reg
 actions+=/variable,name=single_target,value=spell_targets.fan_of_knives<2
 actions+=/variable,name=cold_blood_casted,value=1,if=prev_gcd.1.cold_blood
 actions+=/variable,name=priority_rotation,value=priority_rotation
-actions+=/variable,name=not_pooling|debuff.shiv.up|cooldown.thistle_tea.full_recharge_time<20)|(buff.envenom.up&buff.envenom.remains<=2)|energy.pct>=80|fight_remains<=90,value_else=|debuff.shiv.up|cooldown.thistle_tea.full_recharge_time<20)|energy.pct>=80
+actions+=/variable,name=not_pooling,if=(debuff.shiv.up|cooldown.thistle_tea.full_recharge_time<20)|(buff.envenom.up&buff.envenom.remains<=2)|energy.pct>=80|fight_remains<=90,value_else=|debuff.shiv.up|cooldown.thistle_tea.full_recharge_time<20)|energy.pct>=80

```

Comment:
We'll still need to revisit this.  There's no `value` being set here, and `value_else` starts with an invalid "or' `|`.  What is this variable intended to be?  Is it meant to always equal `(debuff.shiv.up|cooldown.thistle_tea.full_recharge_time<20)|(buff.envenom.up&buff.envenom.remains<=2)|energy.pct>=80|fight_remains<=90`?  If so, the `if` should be `value` and the `,value_else=...` should be cut.